### ISSUE TITLE
version 0.32.0 isn't available on the bazel mirror. 

### ISSUE THREAD

**Issue by charlesoconor** [bazelbuild/rules_go#3170]
### What version of rules_go are you using?

v0.32.0

### What version of gazelle are you using?

v0.25.0

### What version of Bazel are you using?

5.1.1

### Does this issue reproduce with the latest releases of all the above?

v0.32.0 has this issue but v0.31.0 doesn't 

### What operating system and processor architecture are you using?

Tested on both x86 mac and x86 Linux ubuntu 20

### What did you do?

When booting up bazel will report an error that it can't load `rules_go` off of the bazel mirror from 

```
WARNING: Download from https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.32.0/rules_go-v0.32.0.zip failed: class java.io.FileNotFoundException GET returned 404 Not Found
```

`curl https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.32.0/rules_go-v0.32.0.zip` returns a 404 

### What did you expect to see?

Expected no warning. 

### What did you see instead?

Expected there to be no error or the mirror would come back eventually.

**Comment by fmeum** [comment]
Filed https://github.com/bazelbuild/bazel/issues/15617, not sure what went wrong there.
@achew22

**Comment by fmeum** [comment]
Should have been fixed by https://github.com/bazelbuild/bazel/issues/15617, please open a new issue if that's not true.

**Comment by charlesoconor** [comment]
Working now thank you!
